docs/concepts/implementations.md

```diff
@@ -0,0 +1,32 @@
+---
+title: IPSF implementations explained
+description: Learn about the principles that define what an IPFS implementation is.
+---
+
+# IPFS implementations explained
+
+Software, written in any programming language, is considered to be an _IPFS implementation_ if it adheres to the core principles listed below. However, there are characteritics that  characteristics that an IPFS implementation _may_ have, but are not strictly _required_. IPFS principles are broad by design because, like HTTP, IPFS supports an open-ended set of use cases and is adaptable to a broad array of operating conditions. Some implementations are optimized for specific use cases or devices, or use different subsystems to handle content-addressed data. There are multiple specifications in IPFS for handling content-addressed data, and not all implementations implement them.
```

Comment:
```suggestion
Software, written in any programming language, is considered to be an _IPFS implementation_ if it adheres to the core principles listed below. However, there are characteritics  that an IPFS implementation _may_ have, but are not strictly _required_. IPFS principles are broad by design because, like HTTP, IPFS supports an open-ended set of use cases and is adaptable to a broad array of operating conditions. Some implementations are optimized for specific use cases or devices, or use different subsystems to handle content-addressed data. There are multiple specifications in IPFS for handling content-addressed data, and not all implementations implement them.
```

Reply:
Thanks, addressed in https://github.com/ipfs/ipfs-docs/pull/1697/commits/ec97217d73a1076a8127cb3c28151d6e0e831476